### PR TITLE
OOP version of Tiny JS World

### PR DESCRIPTION
**Pull request by ArthurGorbenko**
### OOP Tiny Js World

[Demo](https://arthurgorbenko.github.io/Tiny-JS-World-OOP/) |
[Code base](https://github.com/arthurgorbenko/Tiny-JS-World-OOP/blob/gh-pages/index.js)

The code is submitted in its own sub-directory and in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by ArthurGorbenko** [comment]
@aymkin Thank's ;) Your comments are always substantive and helpful.

File: submissions/ArthurGorbenko/OOP Tiny JS World/index.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+class inhobitant{
```
Comment (aymkin) [inline submissions/ArthurGorbenko/OOP Tiny JS World/index.js]: Class names should start with a capital letter

File: submissions/ArthurGorbenko/OOP Tiny JS World/index.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  }
+}
+
+class pet extends inhobitant {
+  constructor(name,gender,phrase,isHungry){
+    super(name,gender,phrase);
+    this.isHungry = isHungry;
+  }
+  askForFood(){
+    print("I want to eat!");
```
Comment (aymkin) [inline submissions/ArthurGorbenko/OOP Tiny JS World/index.js]: `print` method should not be used in the Class method, according to the task description it should be used to list the inhabitants

File: submissions/ArthurGorbenko/OOP Tiny JS World/index.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  new human("Alex","male","I am studying frontend at Kottans!","Pizza"),
+  new human("Meggy","female","Kottans - one love!","Ice сream"),
+  new metaHuman("woman-Cat","female","Meow, can i join Kottans?","Invisibilty")
+]
+
+inhobitants.forEach(inhobitant =>
+  print(objectsToString(inhobitant))
+);
+
+console.log(pet.name);
```
Comment (aymkin) [inline submissions/ArthurGorbenko/OOP Tiny JS World/index.js]: `console.log` should not be used in prod

**Comment by ArthurGorbenko** [comment]
I'm always ready to improve my code ;) Please,check changes @aymkin

**Pull request merged**